src/js/scrape-niceday.js

```diff
@@ -110,31 +136,8 @@ async function scrapeNiceday() {
       const total_pages = await getTotalPages(page);
 
       for (let j = 1; j <= total_pages; j++) {
-        await page.goto(`${search_urls[i]}&page=${j}`);
-        await autoScroll(page);
-
-        result = await page.evaluate(() => {
-          const items = [...document.querySelectorAll('[class^=CardGellory__StyledProductCard]')];
-
-          const category = document.querySelector('[class^=search__CategoryBannerTitle]').innerText.trim();
-
-          return items.map((item) => {
-            const title = item.querySelector('[class^=ProductCard__Title]').innerText.trim();
-            const description = item.querySelector('[class^=ProductCard__Description]').innerText.trim();
-
-            const link = item.getAttribute('href');
-            const img = item.querySelector('img').getAttribute('src');
-            const price = item.querySelector('[class^=ProductCard__Price]').innerText.trim();
-
-            return {
-              category, title, description, link, img, price,
-            };
-          });
-        });
-
-        storage.push(...result);
-
-        console.log(`page ${j} is done`);
+        const content = await crawlPageContent(page, search_urls[i], j);
```

Comment:
建議
也許可以把 j 改為 pageNum ?
```suggestion
        const content = await crawlPageContent(page, search_urls[i], pageNum);
```